sched/task/task_fork.c

```diff
@@ -137,22 +137,22 @@ FAR struct task_tcb_s *nxtask_setup_fork(start_t retaddr)
 
   /* Allocate a TCB for the child task. */
 
-  child = kmm_zalloc(sizeof(struct task_tcb_s));
+  child = kmm_zalloc(sizeof(struct tcb_s) + sizeof(struct task_group_s));
```

Comment:
This is not the same thing as allocating sizeof(struct task_tcb_s). The correct alignment of task_group_s is not ensured anymore as tcb_s and task_group_s will now be implicitly packed. Is this a problem ?